consul/datadog_checks/consul/consul.py

```diff
@@ -261,24 +262,35 @@ def get_nodes_with_service(self, service):
 
     def _cull_services_list(self, services):
 
+        if self.service_whitelist and self.services_exclude:
+            self.warning('Detected both whitelist and services_exclude options are configured. Consul check will only consider blacklist.')
+            self.service_whitelist = None
+
         if self.service_whitelist:
             if len(self.service_whitelist) > self.max_services:
                 self.warning('More than %d services in whitelist. Service list will be truncated.', self.max_services)
 
             whitelisted_services = [s for s in services if s in self.service_whitelist]
             services = {s: services[s] for s in whitelisted_services[: self.max_services]}
         else:
-            if len(services) <= self.max_services:
+            allowed_services = {s: services[s] for s in services if s not in self.services_exclude}
+            self.warning(str(services) + " " + str(allowed_services))
+
+            self.log.debug('Filtered services {} with service services_exclude {}'.format(services, self.services_exclude))
```

Comment:
```suggestion
            if self.services_exclude:
                self.log.debug('Filtered services {} with service services_exclude {}'.format(services, self.services_exclude))
```
We should log only if there is a `services_exclude` configured so we don't flood the logs with large log lines when not relevant